### PR TITLE
feat: Add CODE_OWNERS

### PR DESCRIPTION
**Pull request by roseDickinson**

### CODE REVIEW

File: CODE_OWNERS [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# owner(s) will be requested for a review.
+# *.js    @js-owner
+
+# ------------------------------------------------------------
+# You can also use email addresses if you prefer. They'll be
+# used to look up users just like we do for commit author
+# emails.
+# *.go docs@example.com
+
+# ------------------------------------------------------------
```
Comment (tg137) [inline CODE_OWNERS]: Just wanted to leave a multi-line comment for banter.

This is cool though

**Review by tg137 — APPROVED** [review]

**Comment by tg137** [comment]
LGTM! This is a cool change and nice that it auto-adds us all!

**Comment by jiphex** [comment]
I think you could potentially create a Github "team" for galileo/pilots/NIMS? and add that to this file, then you could change the people in it (through Github settings) without having to change the repo?

**Comment by roseDickinson** [comment]
Yes that is another possibility

**Review by alex-lee-tessella — APPROVED** [review]
All looks good to me, good thinking using the Galileo team

**Pull request merged**